pkgs/development/interpreters/spidermonkey/78.nix

```diff
@@ -27,6 +27,18 @@ stdenv.mkDerivation rec {
     sha256 = "0451hhjrj9hb6limxim7sbhvw4gs6dd2gmnfxjjx07z3wbgdzwhw";
   };
 
+  patches = []
+    ++ lib.optionals stdenv.hostPlatform.isRiscV [
```

Comment:
Can we apply the patch unconditionally? If it isn't required to be conditional we shouldn't as otherwise it might just "rot" without anyone (that is using `riscv`) noticing.

Reply:
Ok, I've made the patch unconditional and rebased to staging.